editoast/src/views/pathfinding.rs

```diff
@@ -0,0 +1,87 @@
+use actix_web::web::{Data, Json, Path};
+use actix_web::{delete, get};
+use actix_web::{dev::HttpServiceFactory, web, HttpResponse, Responder};
+use editoast_derive::EditoastError;
+use serde::Serialize;
+use thiserror::Error;
+
+use crate::error::Result;
+use crate::models::Pathfinding;
+use crate::models::{Delete, Retrieve};
+use crate::schema::LineString;
+use crate::DbPool;
+
+#[derive(Debug, Error, EditoastError)]
+#[editoast_error(base_id = "pathfinding")]
+enum PathfindingError {
+    #[error("Pathfinding {id} does not exist")]
+    #[editoast_error(status = 404)]
+    NotFound { id: i64 },
```

Comment:
For consistency with other errors.

```suggestion
    NotFound { path_id: i64 },
```